tools/worker-runner/exit/shutdown_unix.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux || darwin || freebsd
```

Comment:
Do we need both forms of the build constraint (or can we delete line 2)?

Note, this comment applies to everywhere there is the syntax change in this PR.

Reply:
Looking at [the new build constraints draft design](https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md) I think we can get away with just the new format, and delete the old lines. I'm not sure if the draft design was replaced with a final version, but I presume there should be no reason to keep both forms.

Reply:
Ah, reading the Compatibility section at the bottom of the page, I see it is intended that both forms are preserved, to enable backward compatibility (presumably so people can continue compiling with an older go version, or importing code as a library). Since we require the newer go version, and this isn't library code that others import, I think it should be safe to remove the lines with the older syntax.